receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -145,7 +151,18 @@ func (jr *jReceiver) tchannelAddr() string {
 	return fmt.Sprintf(":%d", port)
 }
 
-func (jr *jReceiver) AgentCompactThriftAddr() string {
```

Comment:
This function (`AgentCompactThriftAddr()`) was previously public. All other similar functions are private, and this function is being used only in the same source file, so, I changed it to private, as I believe it wasn't public on purpose.